xee/ext.py

```diff
@@ -965,6 +983,10 @@ def open_dataset(
       ee_init_if_necessary: bool = False,
       ee_init_kwargs: Optional[Dict[str, Any]] = None,
       executor_kwargs: Optional[Dict[str, Any]] = None,
+      tile_fetch_kwargs: Dict[str, int] = {
```

Comment:
If you make this argument optional ... you are not required to hard code this dict.